PullRequestClosed/PullRequestClosed.ts

```diff
@@ -5,7 +5,11 @@ export class PullRequestClosed extends PullRequestAction {
     if (this.isEngXpSquad) { // Can't auto-close auto-created issues, the reporter is set to the actual user
       const pr = await this.loadPullRequest(this.payload.pull_request.number);
       if (pr.user.type === "Bot") {
-        await this.jira.moveIssue(issueId, 'Resolve issue', { resolution: { id: this.resolutionId() } });
+        if (issueId.startsWith('PREQ-')) {
+          await this.jira.moveIssue(issueId, 'Resolve issue', { resolution: { id: this.resolutionId() } });
+        } else {
+          await this.jira.moveIssue(issueId, 'Resolve Issue', { resolution: { id: this.resolutionId() } });
+        }
```

Comment:
This doesn't make sense to me. Let's just unify the PREQ workflow to `Resolve Issue`, to follow the style of other names.

I can do the change for you. Can I do it already, or do you need to update some automation on your side too first?

Reply:
Yes please, that's for sure the preferred fix.
Can you check that it is consistent among prod and sandbox? Uppercase I seems to be the standard.

Reply:
Both workflows (prod and sandbox) had the same problem. It was likely an oversight when we were updating them.

I have now updated both to `Close Issue`